### PR TITLE
Hailey

### PR DESCRIPTION
**Pull request by Noitskristopher**

### CODE REVIEW

File: client/src/views/home.js [hunk 1]
```diff
@@ -1,13 +1,76 @@
...
+            <div>
+                <label>How long has it been hurting?</label>
+                <input onChange={(e) = setPainLocation(e.target.value)}
+                value={painDuration}
+                type="text"></input>
+            </div>    <div>
+                <label>Pain level?</label>
+                <input onChange={(e) = setPainLocation(e.target.value)}
+                value={painLevel}
+                type="text"></input>
```
Comment (Noitskristopher) [inline client/src/views/home.js]: change pain level 'type' to 'number'

**Comment by HHarlow** [comment]
fixed the little things!

**Review by HHarlow — COMMENTED** [review]
fixed the little things

**Review by GunnarWalsh — APPROVED** [review]
looks good 👍

**Pull request merged**